src/controllers/FikaClientController.ts

```diff
@@ -25,8 +25,10 @@ export class FikaClientController {
     ) {
         const config = this.fikaConfig.getConfig();
 
-        this.requiredMods = new Set([...config.client.mods.required, "com.fika.core", "com.SPT.custom", "com.SPT.singleplayer", "com.SPT.core", "com.SPT.debugging"]);
-        this.allowedMods = new Set([...this.requiredMods, ...config.client.mods.optional, "com.bepis.bepinex.configurationmanager"]);
+        const sanitizeModList = (strArr: string[]): string[] => strArr.filter(str => str.trim() !== "");
```

Comment:
Could you make this a fully written method? I would rather not have in-line methods.

Reply:
Fixed, but perhaps the method would be better off in a Utils or something rather than the controller.